libs/blocks/hero-marquee/hero-marquee.css

```
@@ -396,6 +439,32 @@ html[dir="rtl"] .hero-marquee li.icon-item span.icon {
   }
 }
 
+/* mweb mobile defaults */
+.hero-marquee.mobile-left-align.center {
+  text-align: left;
```

For RTL purposes, a better declaration might be:
```suggestion
  text-align: start;
```